### ISSUE TITLE
Add AVT coverage for Tearsheet (wide) states: Open/close, disabled, determinate, focus trap, keyboard nav

### ISSUE THREAD

**Issue by elycheea** [carbon-design-system/ibm-products#5521]
- Tearsheet buttons can be disabled (not sure if this needs to be tested separately if it’s based on the button?)
- Tearsheet has a unique stacking behavior (we tagged as “determinate” for lack of a better name/category for now)
